tests/test_cli.py

```diff
@@ -351,15 +351,17 @@ def test_studies_command_flatten(output_format: Optional[str]) -> None:
         if output_format is None or output_format == "table":
             expected_keys_1 = expected_keys_2 = [
                 "name",
+                "user_attrs",
```

Comment:
Thank you for fixing the test.

I think we can add assertions to check if the values of `user_attrs` are correct. For example, we keep `study_1` empty (no user_attrs) and add 
 a user attribute to `study_2`. Then, we check the values in a similar way to `studies[0]["n_trials"]` and `studies[0]["direction"]`.

What do you think?